aixplain/modules/pipeline/designer/enums.py

```diff
@@ -0,0 +1,51 @@
+from enum import Enum
+
+
+class DataType(str, Enum):
+    VIDEO = "video"
+    EMBEDDING = "embedding"
+    TEXT = "text"
+    NUMBER = "number"
+    IMAGE = "image"
+    LABEL = "label"
+    AUDIO = "audio"
+
+
+class RouteType(str, Enum):
+    CHECK_TYPE = "checkType"
+    CHECK_VALUE = "checkValue"
+
+
+class Operation(str, Enum):
+    GREATER_THAN = "greaterThan"
+    GREATER_THAN_OR_EQUAL = "greaterThanOrEqual"
+    LESS_THAN = "lessThan"
+    LESS_THAN_OR_EQUAL = "lessThanOrEqual"
+    EQUAL = "equal"
+    DIFFERENT = "different"
```

Comment:
We may need to add the new `contain` and `notContain` operators.

Reply:
done